concrete/src/Asset/CssAsset.php

```diff
@@ -16,6 +16,13 @@ class CssAsset extends Asset
      */
     protected $assetSupportsCombination = true;
 
+    /**
+     * The default media of this asset.
+     *
+     * @var string
+     */
+    protected $media = 'all';
```

Comment:
What about using `$assetMedia` instead of `$media` (for consistency with other property names)?